plugins/generator-1.19.4/forge-1.19.4/templates/plant/plant.java.ftl

```diff
@@ -391,9 +391,7 @@ public class ${name}Block extends <#if data.plantType == "normal">Flower<#elseif
 </#compress>
 <#-- @formatter:on -->
 
-<#macro canPlaceOnList blockList condition>
+<#macro canPlaceOnList blockList condition blockToCheck>
 <#if (blockList?size > 1) && condition>(</#if>
-<#list blockList as canBePlacedOn>
-groundState.is(${mappedBlockToBlock(canBePlacedOn)})<#sep>||
-</#list><#if (blockList?size > 1) && condition>)</#if>
+${containsAnyOfBlocks(blockList blockToCheck data.excludePlaceableBlocks)}<#if (blockList?size > 1) && condition>)</#if>

```

Comment:
Before this method checked groundState in both cases, but now in second case it checks for blockstate

What was the reason for this change of existing logic? I think it changes condition of canSurvive to not check block below which would not be ok